packages/sanity/src/core/releases/tool/detail/useBundleDocuments.ts

```diff
@@ -67,6 +67,27 @@ export function useBundleDocuments(release: string): {
                 ),
               ),
           ),
+          switchMap((doc) =>
+            observableClient
```

Comment:
Q: doesn't the `doc` already have `_system` on it, as the `doc` is the same doc being queried in `release-documents.check-delete`

Reply:
Hmm might be my lack of knowledge on rxjs but it wasn't showing up when I looked up the doc :/

Reply:
From experimenting with deleting this new `switchMap` I see that the unpublish action badge still appears correctly, so I think `doc` and `document` are the same therefore. I could be missing a not immediately apparent case where this is needed though...? Worth having a look to see if it's necessary though

Reply:
Huh 🤔 
I think the main reason why I needed it wasn't for the showing of the label but being able to enable / disable actions.
However let me have a look 👍

Reply:
huh colour me shocked, for some reason I really though I'd need this for the documentActions aspects. 🤔 
In any case, thank you for pointing this out 🙏